set #pragma attribute after including arm_neon.h, so the attribute only
applies to the local code

Index: mbedtls/library/aesce.c
--- mbedtls/library/aesce.c.orig
+++ mbedtls/library/aesce.c
@@ -46,6 +46,8 @@
 
 #include "aesce.h"
 
+#include <arm_neon.h>
+
 #if defined(MBEDTLS_HAVE_ARM64)
 
 #if !defined(__ARM_FEATURE_AES) || defined(MBEDTLS_ENABLE_ARM_CRYPTO_EXTENSIONS_COMPILER_FLAG)
@@ -66,8 +68,6 @@
 #       error "Only GCC and Clang supported for MBEDTLS_AESCE_C"
 #   endif
 #endif /* !__ARM_FEATURE_AES || MBEDTLS_ENABLE_ARM_CRYPTO_EXTENSIONS_COMPILER_FLAG */
-
-#include <arm_neon.h>
 
 #if defined(__linux__)
 #include <asm/hwcap.h>
